src/emoji/Provider_new.cpp

```diff
@@ -9,7693 +9,7693 @@
 using namespace emoji;
 const QVector<Emoji> emoji::Provider::emoji = {
   // People
-  Emoji{QString::fromUtf8("\xf0\x9f\x98\x80"), "grinning face", emoji::EmojiCategory::People},
+  Emoji{QString::fromUtf8("\xf0\x9f\x98\x80"), "grinning face", emoji::Emoji::Category::People},
```

Comment:
This is actually a generated file, so you need to update scripts/emoji_codegen.py as well!